### PR TITLE
Replace PNG and GIF files with SVG URIs

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
Magnifier:
before-after

Throbber:
before-after-throbber

|     |      Size before      |      Size after     |
|----------|:-------------:|------:|
| Magnifier |  490 byte | 259 byte |
| Throbber |  4.831 byte | 972 byte |

### CODE REVIEW

File: Control.Geocoder.css [hunk 1]
```diff
@@ -60,7 +60,7 @@
 }
 
 .leaflet-control-geocoder-throbber .leaflet-control-geocoder-icon {
-  background-image: url(images/throbber.gif);
+  background-image: url(images/throbber.svg);
```
Comment (simon04) [inline Control.Geocoder.css:63]: The `images/throbber.svg` file has not been committed?

File: Control.Geocoder.css [hunk 2]
```diff
@@ -48,7 +48,7 @@
   height: 26px;
   border: none;
   background-color: white;
-  background-image: url(images/geocoder.png);
+  background-image: url(images/geocoder.svg);
```
Comment (simon04) [inline Control.Geocoder.css:51]: We could (should?) even embed the SVG as data URL here, see https://css-tricks.com/lodge/svg/09-svg-data-uris/. What do you think?

**Review by simon04 — COMMENTED** [review]
This looks very promising! Please see my two comments…

File: Control.Geocoder.css (hunk 1 above)
Comment (KristjanESPERANTO) [reply Control.Geocoder.css:63]: You're right. Now I've committed the file :slightly_smiling_face:

File: Control.Geocoder.css (hunk 2 above)
Comment (KristjanESPERANTO) [reply Control.Geocoder.css:51]: Yes, we can try this. I will change my PR.

Comment (KristjanESPERANTO) [reply Control.Geocoder.css]: I changed it to data URLs.

**Comment by simon04** [comment]
The build fails because there are no more images to copy. To fix, …

```diff
diff --git a/package.json b/package.json
index ad79309..e0b1e3f 100644
--- a/package.json
+++ b/package.json
@@ -6,10 +6,9 @@
   "module": "src/index.js",
   "scripts": {
     "prepare": "npm run build",
-    "build": "npm run build:js && npm run build:css && npm run build:img && npm run build:demo-esbuild && npm run build:demo-rollup && npm run build:demo-webpack",
+    "build": "npm run build:js && npm run build:css && npm run build:demo-esbuild && npm run build:demo-rollup && npm run build:demo-webpack",
     "build:js": "rollup -c",
     "build:css": "cpr Control.Geocoder.css dist/Control.Geocoder.css --overwrite",
-    "build:img": "cpr images/ dist/images/ --overwrite",
     "build:demo-esbuild": "cd demo-esbuild && npm install && npm run build",
     "build:demo-rollup": "cd demo-rollup && npm install && npm run build",
     "build:demo-webpack": "cd demo-rollup && npm install && npm run build",
```

File: Control.Geocoder.css [hunk 3]
```diff
@@ -48,7 +53,7 @@
   height: 26px;
   border: none;
   background-color: white;
-  background-image: url(images/geocoder.png);
+  background-image: var(--magnifier);
```
Comment (simon04) [inline Control.Geocoder.css:56]: To not loose [IE11 support](https://caniuse.com/css-variables), please specify the `url` directly on `background-image`.

Comment (KristjanESPERANTO) [reply Control.Geocoder.css:56]: I'd like to change that, but first I'd like to point out that microsoft's official support for IE11 will end this month. So in my opinion it is time to stop worrying about this browser. Why should we continue to support it when the manufacturer doesn't do it anymore.

Comment (simon04) [reply Control.Geocoder.css:56]: I know. In the past, we've got some bug reports of broken IE11 compatibility. In this case it does make a difference, and even saves a few lines. :)

Comment (KristjanESPERANTO) [reply Control.Geocoder.css:56]: Okay, I'll change that 🙂

Comment (KristjanESPERANTO) [reply Control.Geocoder.css]: Done.

**Comment by KristjanESPERANTO** [comment]
> The build fails because there are no more images to copy. To fix, …

I have adapted the `package.json`.

**Review by simon04 — APPROVED** [review]
Awesome, thank you so much for this PR!

**Comment by KristjanESPERANTO** [comment]
My pleasure. I am pleased that my changes have been accepted :slightly_smiling_face:

**Pull request merged**